arviz/plots/bpvplot.py

```diff
@@ -69,12 +69,12 @@ def plot_bpv(
         How to compute the distributions used as reference for u_values or p_values. Allowed values
```

Comment:
A lot of the parameters above are missing `, optional`. And `data` uses `az.InferenceData` which should not be used, `InferenceData` should be used instead as per https://github.com/arviz-devs/arviz/wiki/Docstring-update-guidelines